editorconfig.el

```diff
@@ -291,6 +291,8 @@ NOTE: Only the **buffer local** value of VARIABLE will be set."
 ;;;###autoload
 (add-hook 'find-file-hook 'edconf-find-file-hook)
 
+(add-to-list 'auto-mode-alist '("\\.editorconfig" . conf-unix-mode))
```

Comment:
Precisely because of that. But it's okay that it will highlight some arbitrary unlikely such filenames regexp matching this, like `settings.editorconfig.txt`.